The version information hasn't been updated by the project and thus generates old paths.

--- multitail-7.1.4/version.orig	2024-08-11 08:58:01.000000000 +0200
+++ multitail-7.1.4/version	2024-08-13 12:05:34.021489656 +0200
@@ -1 +1 @@
-VERSION=7.1.3
+VERSION=7.1.4
